lib/client.ts

```diff
@@ -32,7 +33,7 @@ export default class Client {
     return this.post(URL.reply, {
       messages: toArray(messages),
       replyToken,
-    });
+    }).then(this.checkJSON);
```

Comment:
We needn't care about response here (anyway the response will be empty, thus return type is `any`), so just remove the check.